stellargraph/layer/rgcn.py

```diff
@@ -480,7 +533,7 @@ def __call__(self, x):
 
         return h_layer
 
-    def node_model(self):
+    def _node_model(self):
```

Comment:
Currently all our methods have `node_model` and `link_model` so it would be good to talk in the new team about if these methods in the other classes should be deprecated .. for now just be aware that this makes RGCN only have the new `build` API.